src/components/FormTracker/index.js

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { number, string } from 'prop-types';
 import firebase from '../../firebase';
 
 export const FormTracker = ({ age, email, id, name }) => {
```

Comment:
Yeh `FormTracker` is a bit confusing because it's actually just 1 user.

Reply:
yeah going to change it to UserList to make more sense. And yeah exactly I tried to pass `key` down but that didn't work so had to use `id` instead.

Reply:
isn't this just one user rather than a list of users?

Reply:
yeah actually you make a good point - maybe I should just call it UserCard?